packages/rke2-multus/charts/templates/dhcp-daemonSet.yaml

```
@@ -0,0 +1,62 @@
+---
+{{- if .Values.manifests.dhcpDaemonSet }}
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: multus-dhcp-ds
```

```suggestion
  name: {{ .Release.Name }}-ds
```
(nit) add release name ?

---

I would also nit on putting -ds on the end; we know it's a daemonset. There's no need to use hungarian notation here.

---

done!